File: counsel-server/src/main/java/com/example/counselserver/dto/CounselResponseDto.java
package com.example.counselserver.dto;

import com.example.counselserver.constants.CommunicationStatus;
import lombok.*;

import java.time.LocalDateTime;

@NoArgsConstructor
@AllArgsConstructor
@Builder
@Getter
@Setter
public class CounselResponseDto {
    private Long counselId;
    private String firstname;
    private String lastname;
    private String cellPhone;
    private String email;
    private String memo;
    private String address1;
    private String address2;
    private String city;
    private String state;
    private String zipCode;
    private LocalDateTime appliedAt;
    private CommunicationStatus communicationStatus;
    private LocalDateTime createdAt;
    private String createdBy;
    private LocalDateTime updatedAt;
    private String updatedBy;
}

File: ms-loan-frontend/src/app/dtos/counsel-response-dto.ts
import { CommunicationStatus } from './communitation-status';

export interface CounselResponseDto {
  counselId?: number;
  firstname?: string;
  lastname?: string;
  cellPhone?: string;
  email?: string;
  memo?: string;
  address1?: string;
  address2?: string;
  city?: string;
  state?: string;
  zipCode?: string;
  appliedAt?: string;
  communicationStatus?: CommunicationStatus;
  createdAt?: string;
  createdBy?: string;
  updatedAt?: string;
  updatedBy?: string;
}
